app/components/forms/admin/settings/system-form.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
```

Comment:
@niranjan94 I had mentioned in the PR that form validation will be handled in a separate issue, as this was taking some time, I will add it here itself then.